lms/djangoapps/certificates/models.py

```diff
@@ -108,6 +109,37 @@ class CertificateWhitelist(models.Model):
     user = models.ForeignKey(User)
     course_id = CourseKeyField(max_length=255, blank=True, default=None)
     whitelist = models.BooleanField(default=0)
+    created = CreationDateTimeField(_('created'))
+    free_text = models.TextField(default='')
+
+    @classmethod
+    def certificate_white_list(cls, course_id):
```

Comment:
name of this method is same as the model it is defined in. Can we use a different name here?